src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -84,12 +85,12 @@ describe('Exam assessment', () => {
 
         it('Assess a programming exercise submission (MANUAL)', () => {
             cy.login(tutor, '/course-management/' + course.id + '/exams');
-            cy.contains('Assessment Dashboard', { timeout: examDuration }).click();
+            examManagement.openAssessmentDashboard(exam.id, 120000);
             startAssessing();
             examAssessment.addNewFeedback(2, 'Good job');
             examAssessment.submit();
             cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
-            cy.get('.question-options').contains('6.6 of 10 points').should('be.visible');
+            cy.get('#result-score').should('contain.text', '6.6 of 10 points');
```

Comment:
I think its still important to verify that the score is actually shown to the user. The element could contain the text, but not be visible.
```suggestion
            cy.get('#result-score').should('contain.text', '6.6 of 10 points').and('be.visible');
```